topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorTest.java

```diff
@@ -334,6 +345,15 @@ public void testOnTopicCreated(VertxTestContext context) {
             mockK8s.assertExists(context, resourceName);
             Topic t = TopicSerialization.fromTopicMetadata(topicMetadata);
             mockTopicStore.assertContains(context, t);
+            context.verify(() -> {
+                MeterRegistry registry = metrics.meterRegistry();
+
+                assertThat(registry.get(TopicOperator.METRICS_PREFIX + "resource.state")
+                        .tag("kind", "KafkaTopic")
+                        .tag("name", topicName.asKubeName().toString())
```

Comment:
For all occurences of `topicName.asKubeName().toString()` - This should be enough: `topicName.toString()`.